src/main/scala/smack/messaging/xmmp-extensions/MAMResponse.scala

```diff
@@ -0,0 +1,45 @@
+package smack.scala.extensions
+
+import org.jivesoftware.smack.packet.{ Message, ExtensionElement, IQ }
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+
+// implementation of XEP-0313 Message Archive Management, for querying XMMP server for historical messages
+// see http://xmpp.org/extensions/xep-0313.html
+
+object MAMResponse extends ExtensionInfoProvider {
+  val XmlNamespace = "urn:xmpp:mam:0"
+  val ElementName = "result"
+
+  val ForwardedNamespace = "urn:xmpp:foward:0"
```

Comment:
`foward`... typo in namespace? does this work? or is that the reason why the below is commented out?